app/components/primer/timeline_item_component.html.erb

```diff
@@ -1,17 +1,5 @@
-<%= render Primer::BaseComponent.new(**system_arguments) do %>
-  <% if avatar %>
-    <%= render Primer::AvatarComponent.new(alt: avatar.alt, src: avatar.src, size: avatar.size, square: avatar.square, **avatar.system_arguments) %>
-  <% end %>
-
-  <% if badge %>
```

Comment:
Do we need to maintain the `if` logic in the new approach?

Reply:
we don't! because if the user does not call the slot it will be `nil` and then `<%= nil %>` is noop